priv/stats_descriptions.cfg

```diff
@@ -51,8 +51,8 @@
     {desc, <<"number of _local document write operations">>}
 ]}.
 {[couchdb, httpd, bulk_requests], [
-    {type, counter},
-    {desc, <<"number of bulk requests">>}
+    {type, histogram},
+    {desc, <<"distribution of bulk request sizes">>}
```

Comment:
Let's keep counter in additional to histogram. This shouldn't hurt, but there nobody would be confused about which field to use. Also, `n` here wouldn't be the same as old counter in case of invalid request.